gateways/stripe/stripe_vars.php

```diff
@@ -2,20 +2,46 @@
 
 function espresso_display_stripe($payment_data) {
 	extract($payment_data);
-	global $org_options;
+	global $org_options, $wpdb;
 	$stripe_settings = get_option('event_espresso_stripe_settings');
 	if ($stripe_settings['force_ssl_return']) {
 		$home = str_replace('http://', 'https://', home_url());
 	} else {
 		$home = home_url();
 	}
 	
+	$SQL = "SELECT start_date FROM " . EVENTS_ATTENDEE_TABLE . " WHERE registration_id='%s' ORDER BY id LIMIT 1";
```

Comment:
I'm pretty sure you don't need to put %s in quotes. `$wpdb->prepare` will do that for you (you may actually be unquoting the quote)